src/services/static-snapshot-service.ts

```diff
@@ -1,20 +1,100 @@
+import * as bodyParser from 'body-parser'
+import * as cors from 'cors'
+import * as express from 'express'
+import {Server} from 'http'
+import * as puppeteer from 'puppeteer'
+import * as walk from 'walk'
 import logger from '../utils/logger'
+import {agentJsFilename} from '../utils/sdk-utils'
 import {StaticSnapshotOptions} from './static-snapshot-options'
 
+// Use this instead of importing PercyAgent - we only want the browserified version
```

Comment:
Technically it's rollup now :p Might be better to say "compiled" or "transpiled". Or "babelified"? 🙃